app/generic/verifier/verifier.py

```diff
@@ -246,9 +238,12 @@ async def accept_proof_request(
     try:
         prover = __get_verifier_by_version(presentation.proof_id)
 
-        await assert_valid_prover(
-            aries_controller=aries_controller, prover=prover, presentation=presentation
-        )
+        if not presentation.connectionless:
+            await assert_valid_prover(
```

Comment:
Althoug we can't check the connection, we _could_ check whether the proof request is valid in terms of the schema check. the assert_valid_prover does two checks